### PR TITLE
Upgrade operator version to 112

### PR DESCRIPTION
**Pull request by jaronoff97**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,3 +92,19 @@ define get-crd
 @sed -i 's#\(.*\)- v1beta1#&\n{{- end }}#' $(1)
 @echo '{{- end }}' >> $(1)
 endef
+
+# get-crd doesn't work on macos because of sed incompatibility
+# when upgrading the CRDs on MacOS, swap usage of get-crd to this macro
+define get-crd-macos
```
[the remark this answers is not in the document]
Comment (TylerHelmuth) [reply Makefile:98]: Every time I bump the operator version I actidentally commit my `gsed` changes lol

File: charts/opentelemetry-operator/values.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ manager:
     tag: ""
   collectorImage:
     repository: ""
-    tag: 0.111.0
+    tag: 0.112.0
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/values.yaml:45]: FYI there is a bug with collector 0.112.0 and the hostmetricsreceiver on k8s: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35990

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (jaronoff97) [reply Makefile:98]: yeah oop :( so this one is for us!

File: charts/opentelemetry-operator/values.yaml (hunk 2 above)
Comment (jaronoff97) [reply charts/opentelemetry-operator/values.yaml:45]: argh, i assume there won't be a patch for that right? Should i call this out in the release notes or something?

Comment (TylerHelmuth) [reply charts/opentelemetry-operator/values.yaml:45]: the fix will be in v0.113.0

Comment (jaronoff97) [reply charts/opentelemetry-operator/values.yaml:45]: kk we may as well skip then too...

Comment (TylerHelmuth) [reply charts/opentelemetry-operator/values.yaml:45]: Is there a downside to updating this PR to use v0.113.0 of the collector when it releases this week?

Comment (jaronoff97) [reply charts/opentelemetry-operator/values.yaml:45]: nope, we can do that.

**Comment by TylerHelmuth** [comment]
replaced by https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1410 since @jaronoff97 is on PTO

**Pull request closed without merging**